Tools/autotest/apmrover2.py

```diff
@@ -4621,6 +4622,47 @@ def test_poly_fence_object_avoidance_bendy_ruler(self, target_system=1, target_c
             target_component=target_component,
         )
 
+    def rootdir(self):
+        this_dir = os.path.realpath(__file__)
+        return os.path.relpath(os.path.join(this_dir, "../../../"))
+
+    def script_source_path(self, scriptname):
+        return os.path.join(self.rootdir(), "libraries", "AP_Scripting", "examples", scriptname)
+
+    def installed_script_path(self, scriptname):
+        return os.path.join(self.rootdir(), "scripts", scriptname)
+
+    def install_example_script(self, scriptname):
+        source = self.script_source_path(scriptname)
+        dest = self.installed_script_path(scriptname)
+        shutil.copy(source, dest)
+
+    def remove_example_script(self, scriptname):
+        dest = self.installed_script_path(scriptname)
+        try:
+            os.unlink(dest)
+        except IOError:
+            pass
+        except OSError:
+            pass
+
+    def test_scripting_simple_loop(self):
+        ex = None
+        example_script = "simple_loop.lua"
+        try:
+            self.set_parameter("SCR_ENABLE", 1)
```

Comment:
We may want to ensure that `SCR_HEAP_SIZE` is > 41K as well (this is the default), or do we generally assume the vehicle has these things sorta well setup before running tests?